opennms-container/horizon/container-fs/confd/templates/slack.properties.tpl

```diff
@@ -2,11 +2,17 @@
 # DON'T EDIT THIS FILE :: GENERATED WITH CONFD
 #
 
+export OPENNMS_NOTIFD_SLACK_CHANNEL
```

Comment:
I don't think these should be in this file. When I was talking about testing using environment variables I meant as an alternative key source rather than a yaml file. When confd runs it can pull keys out of the ENV to write to the template.

Reply:
@mattixtech I accidentally pushed this state. Sorry about that. 
Right now I have two versions that work. One using the YAML. One using ENV variables. I'm still figuring out how to combine that :joy: